packages/sw-precaching/src/lib/controllers/base-cache-manager.js

```diff
@@ -14,6 +14,7 @@ class BaseCacheManager {
   constructor(cacheName) {
     this._entriesToCache = new Map();
     this._cacheName = cacheName;
+    this._requestWrapper = new RequestWrapper({cacheName: this._cacheName});
```

Comment:
I hadn't considered the use-case of someone wanting to construct/pass their own Requestwrapper to control pages when there's updates made. Given @gauntface might need to spend time designing how best to handle passing both cacheName and a RequestWrapper that already has a different cache name, perhaps this can be deferred to another PR?